src/index.ts

```diff
@@ -44,19 +46,28 @@ const loadMovies = async (searchTerm: string): Promise<void> => {
   }
 }
 
-async function searchMovies(searchTerm: string): Promise<void> {
-  if (searchTerm.length > 0) {
-    await loadMovies(searchTerm)
-  }
-}
-async function keyPressHandler(e: KeyboardEvent): Promise<void> {
+async function searchMovies(e: KeyboardEvent): Promise<void> {
   if (e.key === 'Enter') {
     const searchTerm = movieSearchBox.value.trim()
+    movieSearchBox.value = ''
     movieDisplay.innerHTML = ''
-    await searchMovies(searchTerm)
+
+    if (!searchTerm.length) {
+      e.preventDefault()
+      const errMsg = []
+      errMsg.push('Please enter a Movie')
+
+      if (errMsg.length > 0) {
+        errorElement.innerText = errMsg.join(',')
+        console.error('NO MOVIE ENTERED')
+      }
+    } else {
+      await loadMovies(searchTerm)
+      errorElement.innerHTML = ''
+    }
   }
 }
 
 // On load
```

Comment:
remove this comment. The code below doesn't act "on load", it responds to a keypress. This is a good example of when comments are useful and when they are not. In this case the comment is a negative, because (1) even if the code below was an onload listenter, the comment wouldn't add any value - you can just read the code and the behavior is obvious (2) as the code changes you'd have to remember to keep the comments in sync which doubles your work. In general comments are better for "why" rather than "what".